client/src/app/disclaimer/disclaimer.component.ts

```diff
@@ -0,0 +1,27 @@
+import { Component } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { ModalComponent } from '../modal/modal.component';
+import { Overlay, OverlayConfig } from '@angular/cdk/overlay';
+import { ComponentPortal } from '@angular/cdk/portal';
```

Comment:
I see the imports that are not used. 
If you are not using them, they should be removed

Reply:
thanks, fixed it